app/views/main.scala.html

```diff
@@ -77,8 +78,11 @@
 }
 
 @signOutUrl = @{
-    mustacheCheck {
-        routes.ServiceController.serviceSignout().url
+    mustacheCheck{
+        val signOutLink = routes.ServiceController.serviceSignout().url
+        authedUser.fold{
+            if(isAuthenticated) signOutLink else ""
```

Comment:
If there is an authedUser, it's already authenticated, you don't need this boolean